yarn.lock

```diff
@@ -5174,6 +5174,15 @@
     "@uniswap/v2-core" "1.0.1"
     "@uniswap/v3-core" "1.0.0"
 
+"@uniswap/universal-router@^1.0.1":
```

Comment:
This should be de-duped - just run `yarn deduplicate`. I'd expect it to fail deps-test as-is.

Reply:
`yarn deduplicate` did nothing 🤔